recipes/jupyterlab_widgets/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.0.0a4" %}
+{% set sha256 = "a51f905de826a15bac5aab995e9f49d1dd4b4b8831f54536b7bad36ee318bbea" %}
+
+package:
+  name: jupyterlab_widgets
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/jupyterlab_widgets/jupyterlab_widgets-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  noarch: python
```

Comment:
> Could this package be noarch?

@mariusvniekerk - doesn't this mean the package is noarch? Or is there more that we need to put in somewhere?

Reply:
Nope this is sufficient